services/stats/stats.go

```diff
@@ -94,18 +94,20 @@ type RudderStatsT struct {
 func Setup() {
 	var err error
 	conn = statsd.Address(statsdServerURL)
-	//TODO: Add tags by calling a function...
-	client, err = statsd.New(conn, statsd.TagsFormat(statsd.InfluxDB), defaultTags())
-	if err != nil {
-		// If nothing is listening on the target port, an error is returned and
-		// the returned client does nothing but is still usable. So we can
-		// just log the error and go on.
-		pkgLogger.Error(err)
-	}
-	if client != nil {
-		rruntime.Go(func() {
-			collectRuntimeStats(client)
-		})
+	if !statsEnabled {
```

Comment:
Lets add this in a different PR?

Reply:
https://github.com/rudderlabs/rudder-server/pull/1022/files